hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/MappingDefinition.java

```diff
@@ -14,52 +14,55 @@
  * limitations under the License.
  */
 
-package com.hazelcast.jet.sql.impl.schema;
+package com.hazelcast.jet.sql.impl.connector.infoschema;
 
+import com.hazelcast.jet.sql.impl.schema.Mapping;
+import com.hazelcast.jet.sql.impl.schema.MappingField;
 import com.hazelcast.sql.impl.schema.Table;
 import com.hazelcast.sql.impl.schema.TableField;
 
 import java.util.List;
-import java.util.Map;
 import java.util.Map.Entry;
 import java.util.stream.Collectors;
 
 public class MappingDefinition {
 
     private final Table table;
```

Comment:
I think we can remove this field. `name()` will return `mapping.name()`, `schema()` will return `SCHEMA_NAME_PUBLIC` and we should not need the `tableFields()` after we remove the hidden columns from the `columns` view.